src/constants/index.ts

```diff
@@ -3739,4 +3741,417 @@ export const STORAGE_ABI = [
     "stateMutability": "nonpayable",
     "type": "function"
   }
-] as const
\ No newline at end of file
+] as const
+
+export const BRIDGED_TOKEN_ABI = [
```

Comment:
Rather than using a specific abi, it would be better to use the generic ERC20 abi. 
Especially since the 'bridged token' functions are not intended to be used by users, and here it is only the 'approve' and 'transfer' functions of the ERC20 that are targeted